src/components/Search/Search.tsx

```diff
@@ -1,9 +1,91 @@
-import React from 'react'
+import React, { useState, useReducer } from 'react'
+import { skillsData, valuesData } from '../../assets/test-values-skills'
+import { Redirect } from 'react-router-dom'
+
 import './Search.scss'
 
+const initialState = {
+	skills: skillsData,
+	values: valuesData,
+	runSearch: false
+}
+
+function reducer(state:object, update: {type:string, change:any}) {
```

Comment:
Awesome job getting the functionality of determining which boxes are checked (and also unchecked) working with useReducer. For sake of time, let's run with this way of doing it so we can continue working towards MVP. I will approve the merge!

I am going to checkout to `feature/ge-search+filter` as noted in your PR notes and attempt to get it working a different way. I think we can use the checkboxes still, but the flow will be somewhat different which takes in to account the performance:
1. When a user clicks a checkbox, a method will be called that will:
 - If the box is NOW checked, push the attribute into the corresponding option array
 - If the box is NOW unchecked, remove the attribute from the corresponding option array
2. When the Search button is clicked, it will pass the option arrays to the BE for the filtering to happen.

This will take a little bit more research on my end, but I think it will require less iterating over data. If I find a working solution, I'll bring it to your attention!

Awesome work!